src/reducers/cell-reducer.js

```diff
@@ -48,6 +52,124 @@ const cellReducer = (state = newNotebook(), action) => {
       return nextState
     }
 
+    case 'HIGHLIGHT_CELL': {
+      const cells = state.cells.slice()
+      const index = cells.findIndex(c => c.id === action.id)
+      const thisCell = cells[index]
+      if (action.revert) thisCell.highlighted = !thisCell.highlighted
+      else thisCell.highlighted = true
+      nextState = Object.assign({}, state, { cells })
+      return nextState
+    }
+
+    case 'UNHIGHLIGHT_CELLS': {
+      const cells = state.cells.slice()
+      cells.forEach((c) => { c.highlighted = false }) // eslint-disable-line
```

Comment:
if you're disabling eslint, specify the rule being disabled

Reply:
in this case, you can avoid disabling the eslint and skip the `cells.slice` above using a map (which accomplishes the same copying as the slice). something along the lines of:
```
       const cells = state.cells.map((c) =>Object.assign({}, c, { highlighted: false })) 
```